### PR TITLE
Incomplete Maven Goal descriptions

### PR DESCRIPTION
**Pull request by Dufgui**

### CODE REVIEW

**Comment by geoand** [comment]
Thanks for the PR @dufgui!

I am guessing that CI is failing because of formatting (I'm on a mobile so it's hard to check), so I propose you do `mvn install -DskipTests -Dno-native` to allow formatting to be fixed automatically.

File: devtools/maven/src/main/java/io/quarkus/maven/RemoteDevMojo.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
 /**
  * The dev mojo, that connects to a remote host
  * <p>
+ * You can launch forked app directly with {@code dev}
```
Comment (gsmet) [inline devtools/maven/src/main/java/io/quarkus/maven/RemoteDevMojo.java]: I don't think I would add this sentence. I would just add a final dot to the former one and remove the `<p>`

File: devtools/maven/src/main/java/io/quarkus/maven/NativeImageMojo.java [hunk 2]
```diff
@@ -36,6 +36,14 @@
 import io.quarkus.creator.phase.nativeimage.NativeImagePhase;
 import io.quarkus.creator.phase.runnerjar.RunnerJarOutcome;
 
+/**
+ * Build a native executable of your application.
+ * It improves the startup time of the application, and produces a minimal disk footprint.
+ * The executable would have everything to run the application including the "JVM" (shrunk to be just enough to run the
+ * application), and the application.
+ * <p>
+ * You can build the application before with {@code build}
```
Comment (gsmet) [inline devtools/maven/src/main/java/io/quarkus/maven/NativeImageMojo.java]: I think you should just keep `Build a native executable of your application.`. The rest is too verbose and is not really useful IMHO.

File: devtools/maven/src/main/java/io/quarkus/maven/ExampleConfigMojo.java [hunk 3]
```diff
@@ -38,6 +38,9 @@
 import org.apache.maven.plugins.annotations.ResolutionScope;
 import org.apache.maven.project.MavenProject;
 
+/**
+ * This goal helps in configuring Quarkus Maven project with quarkus-maven-plugin, with an example of configuration file
```
Comment (gsmet) [inline devtools/maven/src/main/java/io/quarkus/maven/ExampleConfigMojo.java]: I would go with just a simple `Creates an example configuration file.`.

**Review by gsmet — CHANGES_REQUESTED** [review]
I added some comments inline. Thanks for the PR.

**Comment by Dufgui** [comment]
> This one actually does more than just "build." It actually creates an uber-jar for the application. Be good to mention that in the description.

just if we use the option: uberJar no ?

**Review by gsmet — APPROVED** [review]

**Pull request merged**